test/_functions/python/timeout/timeout.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2017 The Nuclio Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import time
+import json
+
+
+def handler(context, event):
+    """Wait a timeout amount and return current PID"""
+
+    context.logger.debug('Event body: %r', event.body.decode('utf-8'))
+
+    request = json.loads(event.body)
```

Comment:
You don't need to loads the body, just pass json content-type

Reply:
OK